brainstormforce/astra
# AST-1286 - WPForm submit & normal button tags not inheriting customizer button color styles

### Description
- Due to :not(.components-button) CSS, static color styling applies to all other buttons. 
- Commit - https://github.com/brainstormforce/astra/pull/3513/commits/bc5704071a6ad48fe5d94105a529fd0bb904ab3e where we are added this CSS to restrict editor's compoenent-button getting static this CSS applied

### Screenshots


### Types of changes
- Reverted that CSS & for editor specific CSS, its overridden by parent class selector

### How has this been tested?
- Check any WP Forms submit button
- Check any button HTML elemen
- Check component buttons - https://share.bsf.io/GGu462vP (these should be black in color)

### Checklist:
